Annotato/Annotato/ViewModels/Document/DocumentViewModel.swift

```diff
@@ -200,5 +237,21 @@ extension DocumentViewModel {
 
             self?.receiveDeleteAnnotation(deletedAnnotation: deletedAnnotation)
         }.store(in: &cancellables)
+
+        documentsPersistenceManager.$updatedDocument.sink(receiveValue: { [weak self] updatedDocument in
+            guard let updatedDocument = updatedDocument else {
+                return
+            }
+
+            self?.receiveUpdateDocument(updatedDocument: updatedDocument)
+        }).store(in: &cancellables)
+
+        documentsPersistenceManager.$deletedDocument.sink(receiveValue: { [weak self] deletedDocument in
+            guard let deletedDocument = deletedDocument else {
+                return
+            }
+
+            self?.receiveDeleteDocument(deletedDocument: deletedDocument)
+        }).store(in: &cancellables)
```

Comment:
Is there a need to check if the ids are the same like we do in the annotation subs above?